docs/build-clio.md

```diff
@@ -88,6 +88,31 @@ If successful, `conan install` will find the required packages and `cmake` will
 > [!TIP]
 > To generate a Code Coverage report, include `-o coverage=True` in the `conan install` command above, along with `-o tests=True` to enable tests. After running the `cmake` commands, execute `make clio_tests-ccov`. The coverage report will be found at `clio_tests-llvm-cov/index.html`.
 
+### Generating API docs for Clio
+
+The API documentation for Clio is generated by [Doxygen](https://www.doxygen.nl/index.html). If you want to generate the API documentation when building Clio, make sure to install Doxygen on your system.
+
+To generate the API docs:
+
+1. First, include `-o docs=True` in the conan install command. For example:
```

Comment:
We are assuming the reader knows to make the `build` folder and change the workdir to it here, is that ok?

Reply:
It's because we've already told users to create the `build` folder in the initial section [Building Clio](https://github.com/maria-robobug/clio/blob/1194-add-instructions-for-doxygen/docs/build-clio.md#building-clio)

So I didn't want to repeat that part of the step, but I can add it in again, can't hurt.